MyArrayList/Program.cs

```diff
@@ -74,6 +85,7 @@ public void Sort(bool descending)
             _list.Reverse();
         }
     }
+    
     //finds the maximum value of the list
```

Comment:
It is better to return max value as a result of work of this method. Because in current implementation method cannot be used in other calculations and actualy useless. It is just standalone method that put result in console. It I need to calculate the sum of min and max value of the array I cannot use this method.